mappings/net/minecraft/village/TradeOffers.mapping

```diff
@@ -1,6 +1,20 @@
 CLASS net/minecraft/class_3853 net/minecraft/village/TradeOffers
 	FIELD field_17067 PROFESSION_TO_LEVELED_TRADE Ljava/util/Map;
 	FIELD field_17724 WANDERING_TRADER_TRADES Lit/unimi/dsi/fastutil/ints/Int2ObjectMap;
+	FIELD field_30615 DEFAULT_MAX_USES I
+	FIELD field_30616 MAX_USES_COMMON I
+	FIELD field_30617 MAX_USES_RARE I
+	FIELD field_30618 XP_NOVICE_SELL I
+	FIELD field_30619 XP_NOVICE_BUY I
+	FIELD field_30620 XP_APPRENTICE_SELL I
+	FIELD field_30621 XP_APPRENTICE_BUY I
+	FIELD field_30622 XP_JOURNEYMAN_SELL I
+	FIELD field_30623 XP_JOURNEYMAN_BUY I
+	FIELD field_30624 XP_EXPERT_SELL I
+	FIELD field_30625 XP_EXPERT_BUY I
+	FIELD field_30626 XP_MASTER_TRADE I
+	FIELD field_30627 PRICE_MULTIPLIER_LOW F
+	FIELD field_30628 PRICE_MULTIPLIER_HIGH F
```

Comment:
```suggestion
	FIELD field_30616 COMMON_MAX_USES I
	FIELD field_30617 RARE_MAX_USES I
	FIELD field_30618 NOVICE_SELL_XP I
	FIELD field_30619 NOVICE_BUY_XP I
	FIELD field_30620 APPRENTICE_SELL_XP I
	FIELD field_30621 APPRENTICE_BUY_XP I
	FIELD field_30622 JOURNEYMAN_SELL_XP I
	FIELD field_30623 JOURNEYMAN_BUY_XP I
	FIELD field_30624 EXPERT_SELL_XP I
	FIELD field_30625 EXPERT_BUY_XP I
	FIELD field_30626 MASTER_TRADE_XP I
	FIELD field_30627 LOW_PRICE_MULTIPLIER F
	FIELD field_30628 HIGH_PRICE_MULTIPLIER F
```
Use the word order as in written or spoken language, like "master's trade xp" becomes `MASTER_TRADE_XP` than `XP_MASTER_TRADE`